.github/workflows/sync-rtd-redirects.yaml

```
@@ -40,7 +40,15 @@ jobs:
       - name: Install readthedocs-cli
         run: python3 -m pip install readthedocs-cli
 
-      - name: Sync redirects
+      - if: github.ref == format('refs/heads/{0}', github.event.repository.default_branch)
```

```suggestion
      - if: github.ref_type == 'branch' && github.ref_name == github.event.repository.default_branch
```

This is what I used in [docker-base/ci.yml](https://github.com/nextstrain/docker-base/blob/c4df6aab877d3b213f3716069c76907b328ded0c/.github/workflows/ci.yml#L13).

According to the [`github` context doc](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context), this should be equivalent and I think it expresses the "branch-only" condition more clearly.

---

Nod. I saw that when authoring this and looking at ways of specifying the condition, but I personally preferred the `github.ref == …` condition here as it felt clearer to me that "this runs on a single ref". That said, I might be in the minority in being familiar and comfortable with git's fully-qualified refs.

I (somewhat accidentally) merged this a few minutes ago before seeing the comment here, so I didn't make the change above, but if it seems worth it to you then please feel free to do so.

---

I'm fine with keeping it as-is, this was non-blocking.